docs/cloudformation.json

```diff
@@ -239,7 +239,7 @@
           {
             "LoadBalancerPort": "80",
             "InstancePort": "8080",
-            "Protocol": "HTTP"
+            "Protocol": "TCP"
```

Comment:
One downside with this is that it seems we need to put the ELB in `TCP` mode to be able to use Hijack. If the ELB is in `HTTP` mode, hijacking the connection and attempting to write to it results in a `broken pipe` error.

In prod, this would be `SSL` here. This isn't a big deal for the controllers, since we just lose the proxy headers.